src/ide/inlay_hints/types.rs

```diff
@@ -0,0 +1,62 @@
+use crate::{ide::ty::InferedValue, lang::db::AnalysisDatabase};
+use cairo_lang_semantic::{GenericArgumentId, TypeId, TypeLongId};
+use cairo_lang_syntax::node::{
+    ast::{Expr, GenericArg, GenericArgValue, TerminalUnderscore, UnaryOperator},
+    helpers::PathSegmentEx,
+};
+use cairo_lang_utils::LookupIntern;
+
+pub fn follow_underscores(
```

Comment:
Please rename this and add a comment explaining what fragment this exactly applies to